docs/apm/agent-configuration.asciidoc

```diff
@@ -34,6 +31,26 @@ Kibana communicates any changed settings to APM Server so that your agents only
 [float]
 ==== Supported configurations
 
+[float]
+===== `CAPTURE_BODY`
```

Comment:
I capitalized these because the original config was capitalized. Should they be lower-cased instead? Does it matter?